package.json

```diff
@@ -17,7 +17,8 @@
     "preversion": "npm test",
     "version": "npm run build && git add release/ && npm run update-bower-version && git add bower.json",
     "postversion": "git push && git push --tags",
-    "update-bower-version": "bash scripts/update-bower-version.sh"
+    "update-bower-version": "bash scripts/update-bower-version.sh",
+    "autobuild-docs": "cd doc && sphinx-autobuild . _build/html; cd .."
```

Comment:
The way @gregkare explained it, we don't need to CD back out here, because the script is run in a new shell in a subprocess iirc.

Reply:
My comment was about the Makefile, I don't know how npm run-script behaves

Reply:
I just tried it, going back isn't needed